admin_guide/out_of_resource_handling.adoc

```diff
@@ -116,21 +119,20 @@ To configure hard eviction thresholds, the following flag is supported:
 The node evaluates eviction thresholds per its housekeeping interval. It
 monitors every 10 seconds and the value can not be modified.
 
-* `housekeeping-interval` is the interval between container housekeepings.
+* `housekeeping-interval`: the interval between container housekeepings.
 
 For more information about housekeeping, see the
 link:https://github.com/google/cadvisor/blob/master/docs/runtime_options.md[cAdvisor
 documentation].
```

Comment:
Is cAdvisor something supported by OpenShift? I've not heard of it before. Is there a better place we could link out to?